client/service/src/builder.rs

```diff
@@ -308,7 +308,7 @@ pub fn new_full_parts<TBl, TRtApi, TExecDisp>(
 {
 	let keystore_container = KeystoreContainer::new(&config.keystore)?;
 
-	let telemetry_span = if config.telemetry_endpoints.is_some() {
+	let telemetry_span = if config.telemetry_endpoints().is_some() {
 		Some(TelemetrySpan::new())
 	} else {
 		None
```

Comment:
I think I commented on this on the other PR, [here](https://github.com/paritytech/substrate/pull/7463#discussion_r560895574).

Reply:
Yes I remember... Not sure why I lost it at some point... :woman_shrugging:  Will fix it

Reply:
Done in 783d1dda9